website/docs/cloud-docs/api-docs/team-members.mdx

```diff
@@ -141,3 +190,52 @@ $ curl \
   --data @payload.json \
   https://app.terraform.io/api/v2/teams/257525/relationships/users
 ```
+
+## Delete a User from Team (With organization membership ID)
+
+This method removes multiple users from a team using the organization membership ID. This DOES NOT delete the user; it only removes them from this team.
```

Comment:
```suggestion
This method removes multiple users from a team using the organization membership ID. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.